web/client/components/catalog/editor/AdvancedSettings/WMSDomainAliases.js

```diff
@@ -0,0 +1,76 @@
+import {FormControl, FormGroup, Col, ControlLabel, Glyphicon, Button} from "react-bootstrap";
```

Comment:
add copyright comments for new files with the year updated to current one

Reply:
Added